lib/govuk_publishing_components/presenters/select_with_search_helper.rb

```diff
@@ -0,0 +1,84 @@
+##
```

Comment:
I don't know why we've got this code here nothing seems to be using it? I also don't understand why we'd add code under the GovukPublishingComponents namespace here 😕